package.json

```diff
@@ -4,7 +4,9 @@
   "version": "0.1.2",
   "scripts": {
     "storybook": "start-storybook -p 6006 -s ./public",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "prettier": "prettier --list-different \"**/*.vue\"",
+    "prettier-fix": "prettier --write \"**/*.vue\""
```

Comment:
My goal was to connect Prettier with Webpack, to have this as an integrated part of the workflow, to don't even have to think about it.

Reply:
Check this out - https://medium.com/@doppelmutzi/eslint-prettier-vue-workflow-46a3cf54332f

Will be way better to base prettier on eslint rules, than crate own from scratch.